idea_town/frontend/static-src/app/views/base-view.js

```diff
@@ -33,7 +33,7 @@ export default AmpersandView.extend({
 
     // HACK: Slap the same data-l10n-args data on every localized node, because
     // the most common case is they all need the same model data.
-    const nodes = this.el.querySelectorAll('[data-l10n-id]');
+    const nodes = this.queryAll('[data-l10n-id]');
```

Comment:
It's fine if you want to include this bug fix in this PR, but it needs to be in a separate commit.

There are tons of ways to do this, but here's one way I'd do it:
```sh
# pull this file out of the main commit
git reset HEAD~1 idea_town/frontend/static-src/app/views/base-view.js
# amend the main commit
git commit --amend --no-edit
# create a new commit
git add idea_town/frontend/static-src/app/views/base-view.js
git commit -m "Use an array, not a NodeList, to allow for-of iteration of l10n nodes"
```
From there, you can `git rebase -i HEAD~2` and swap the order of the two commit lines to put the bug fix earlier in history than the main testing commit.